documentation/modules/oauth/con-oauth-config-keycloak.adoc

```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+//
+// assembly-oauth.adoc
+
+[id='con-oauth-strimzi-config-keycloak-{context}']
+= Keycloak authorization server
+
+You can use Keycloak as an {oauth} authorization server for {ProductName}.
```

Comment:
Can we make this sound less like recomendation? Either mention other servers as well (I think @mstruk tested at least Ory Hydra). Alternatively, we can mention that Keycloak is what we use primarily for testing.

Reply:
I've made the section more generic and mentioned Keycloak and Hydra as examples, with links